<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QWidget" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pbBack">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>690</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>1000</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>650</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pbConnect">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>690</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDisconnect">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>690</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditMessage">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>650</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="leIP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
   <property name="text">
    <string>127.1.0.1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
